### PR TITLE
Tweak eni filters to search for a particular keyword instead

### PR DESCRIPTION
**Pull request by limed**

### CODE REVIEW

**Comment by limed** [comment]
Fixes issue #6

File: nubis/puppet/files/eni-associate [hunk 1]
```diff
@@ -38,7 +38,7 @@ function _get_eni_id {
     aws ec2 describe-network-interfaces \
         --region ${region} \
         --filter \
-            Name=tag-value,Values=${stack_name}\
+            Name=tag-value=Values=nubis-nat-eni \
```
Comment (tinnightcap) [inline nubis/puppet/files/eni-associate]: is the change from a coma to an equals between the k/v pairs intentional?

Comment (limed) [inline nubis/puppet/files/eni-associate]: Whups thats my bad, its a typo. I'll fix in next commit

**Comment by tinnightcap** [comment]
r+

**Pull request merged**